src/main/scala/is/hail/methods/KinshipMatrix.scala

```diff
@@ -11,7 +13,7 @@ import is.hail.utils._
 /**
   * Represents a KinshipMatrix. Entry (i, j) encodes the relatedness of the ith and jth samples in sampleIds.
   */
-class KinshipMatrix(val hc: HailContext, val matrix: IndexedRowMatrix, val sampleIds: Array[String]) {
+class KinshipMatrix(val hc: HailContext, val matrix: IndexedRowMatrix, val sampleIds: Array[String], val numVariantsUsed: Long) {
```

Comment:
I don't care that much about this, but still think it's the right design decision. If KinshipMatrix gets extended, we'll probably want to make it a case class just for the `copy` method. Since a bunch of the parameters check reference equality, the case class equality would only be satisfied by doing something like `new KM(hc, matrix, sampleIds, nVariants) == this` or something, but that seems okay. I think the toString with the four members is totally fine as well -- better than the class name + memory location alone, for sure